builtin/mainmenu/tab_settings.lua

```diff
@@ -71,6 +71,24 @@ local function video_driver_fname_to_name(selected_driver)
 	return ""
 end
 
+local default_label = fgettext("Default")
```

Comment:
Why did you define this variable instead of using `fgettext("Default")` directly?